service/lib/php/ext/thrift_protocol/php_thrift_protocol.cpp

```diff
@@ -429,7 +429,7 @@ void binary_deserialize(int8_t thrift_typeID, PHPInputTransport& transport, zval
         throw_tprotocolexception(errbuf, INVALID_DATA);
         RETURN_NULL();
       }
-      binary_deserialize_spec(return_value, transport, Z_ARRVAL_P(spec));
+      binary_deserialize_spec(return_value, transport, Z_ARRIVAL_P(spec));
```

Comment:
have you regenerated the thrift interfaces? I'm not sure if this is coming from Hive

Reply:
oops, thanks, this is just a botch ARRVAL is presumably short for array-val. dropping it and rebasing ...